test/acceptance/replication/async_replication/async_repair_test.go

```diff
@@ -145,38 +145,39 @@ func (suite *AsyncReplicationTestSuite) TestAsyncRepairSimpleScenario() {
 
 	t.Run("restart node 3", func(t *testing.T) {
 		common.StartNodeAt(ctx, t, compose, 3)
-		time.Sleep(time.Second)
-	})
-
-	// wait for some time for async replication to repair missing object
-	time.Sleep(3 * time.Second)
-
-	t.Run("stop node 2", func(t *testing.T) {
-		common.StopNodeAt(ctx, t, compose, 2)
 	})
 
 	t.Run("assert new object read repair was made", func(t *testing.T) {
-		resp, err := common.GetObjectCL(t, compose.GetWeaviateNode(3).URI(),
-			repairObj.Class, repairObj.ID, replica.One)
-		require.Nil(t, err)
-		assert.Equal(t, repairObj.ID, resp.ID)
-		assert.Equal(t, repairObj.Class, resp.Class)
-		assert.EqualValues(t, repairObj.Properties, resp.Properties)
-		assert.EqualValues(t, repairObj.Vector, resp.Vector)
+		assert.EventuallyWithT(t, func(ct *assert.CollectT) {
+			resp, err := common.GetObjectCL(t, compose.GetWeaviateNode(3).URI(),
```

Comment:
shouldn't we use `ct` here instead of `t`?

Reply:
also checked that, the datatype is actually different and common methods make use of the underlying datatype of t